serve-web/tests/Service/ReportServiceTest.php

```diff
@@ -153,8 +153,7 @@ protected function numberOfOrdersDataProvider()
     {
         return [
             'tenOrders' => [10],
-            'tenThousandOrders' => [10000],
-            'thirtyThousandOrders' => [30000],
```

Comment:
Why did you remove the larger numbers of orders? These were put in to load test report generation. If they aren't working with your changes, we'll need to think again about how to do it.